Docs/uri.md

```diff
@@ -0,0 +1,23 @@
+# URI
+
+The `URI` type represents an URI.
+
+```swift
+public struct URI {
+    public var scheme: String?
+    public var userInfo: (username: String, password: String)?
+    public var host: String?
+    public var port: Int?
+    public var path: String?
+    public var query: [(key: String, value: String?)]
```

Comment:
What's the situation where `value` will be `nil` if `key` has value? Would it be something like `key=` w/o a value?